src/Data/Profunctor/Unsafe.hs

```diff
@@ -227,11 +244,15 @@ instance Functor f => Profunctor (Joker f) where
   dimap _ g (Joker fb) = Joker (fmap g fb)
   {-# INLINE dimap #-}
 
-instance (Profunctor p, Functor f, Functor g) => Profunctor (Biff p f g) where
+instance (Profunctor p, Functor f, Functor g, Bifunctor p) => Profunctor (Biff p f g) where
```

Comment:
@chshersh the addition of `Bifunctor` here is not correct, could you please remove it?

Reply:
Similarly, the use of `Bifunctor` here is forced by the instance context in the `Functor` instance for `Biff`.